src/libraries/Native/Unix/System.Native/pal_networking.c

```diff
@@ -56,6 +56,10 @@
 #if HAVE_LINUX_CAN_H
 #include <linux/can.h>
 #endif
+#if HAVE_LINUX_AIO
+#include <linux/aio_abi.h>
```

Comment:
important: the AIO is available on Linux, but not on all Unixes (like MacOS), hence I needed this switch

Reply:
AIO is Linux only.  No other OS will have this. Using `#if defined(__linux)` is also an option.

Reply:
I guess we do not support kernels where `linux/aio_abi.h` is not available, right?